x-pack/plugins/observability_ai_assistant/common/functions/prompt_suggestions.ts

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { FromSchema } from 'json-schema-to-ts';
+
+export const promptSuggestionsFunctionDefinition = {
+  name: 'prompt_suggestions',
+  contexts: ['core' as const],
+
+  description:
+    'Use this function to suggest questions that the user can ask to continue the conversation. Call this for every message.',
```

Comment:
I cannot get the LLM to call the function at all - even when adding "Call this for every message".

Reply:
Have you tried adding it in the system message?

Reply:
Yes see below

> Use the "prompt_suggestions" function to suggest prompts that the user can ask to continue the conversation. Call this function after every message.

Reply:
Try something stronger, like "YOU MUST CALL THIS FUNCTION if you don't call any others"

Reply:
I love this new paradigm of programming. We are at a point where yelling at the machine is actually helpful